subprojects/shipkit/src/test/groovy/org/shipkit/internal/notes/format/DetailedFormatterTest.groovy

```diff
@@ -52,18 +54,19 @@ Release notes:
 
 Release notes:
 
-**2.0.0 (2017-01-04)** - [1 commit](http://commits/v1.9.0...v2.0.0) by Szczepan Faber - published to [![Bintray](https://img.shields.io/badge/Bintray-2.0.0-green.svg)](Bintray/2.0.0)
+#### 2.0.0
+ - (2017-01-04) [1 commit](http://commits/v1.9.0...v2.0.0) by Szczepan Faber - published to [![Bintray](https://img.shields.io/badge/Bintray-2.0.0-green.svg)](Bintray/2.0.0)
  - No pull requests referenced in commit messages."""
     }
 
     def "formats header when emphasized version"() {
         expect:
-        DetailedFormatter.header("v0.1.0", new Date(1483500000000), false) == "**v0.1.0 (2017-01-04)** - "
+        DetailedFormatter.header("v0.1.0", false) == "#### v0.1.0"
     }
 
     def "formats header when regular version"() {
         expect:
-        DetailedFormatter.header("v0.1.0", new Date(1483500000000), true) == "# v0.1.0 (2017-01-04) - "
+        DetailedFormatter.header("v0.1.0", true) == "# v0.1.0"
```

Comment:
I guess now emphasized version would be "#", and a regular one "####". What do you think @mockitoguy ?